File: Leetcode/1295/Linear_Search_Algo/main.py
# using python old
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        for num in nums:
            
            num = int(math.log10(num) + 1)
            if (num%2==0):
                count = count + 1
        return count

        

File: Leetcode/1295/Linear_Search_Algo/Main.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for (int num : nums) {
            num = (int) (Math.log10(num) + 1);          // using (int) Math.log10(x) + 1 number of digit number have
            if (num%2==0) count++;
        }
        
        return count;
    }
}